tests/notifier/notifier_test.py

```diff
@@ -120,5 +120,140 @@ def test_notifications_for_nonempty_violations(
         self.assertEquals(1, mock_gcs_violations.run.call_count)
 
 
+class InventorySummaryNotifierTest(ForsetiTestCase):
+    """Tests for the inventory summary notifier."""
+
+    def setUp(self):
+        """Setup."""
+        ForsetiTestCase.setUp(self)
+        self.fake_utcnow = datetime(year=1920, month=5, day=6, hour=7, minute=8)
+
+    def tearDown(self):
+        """Tear down method."""
+        ForsetiTestCase.tearDown(self)
+
+    @mock.patch(
+        'google.cloud.forseti.notifier.notifier.run_inv_summary',
+        autospec=True)
+    @mock.patch(
+        'google.cloud.forseti.notifier.notifier.find_notifiers', autospec=True)
+    @mock.patch(
+        'google.cloud.forseti.notifier.notifier.scanner_dao', autospec=True)
+    def test_no_notifications_for_empty_violations(
```

Comment:
This is not the correct name or doc for this test.  You want something like `test_inventory_summary_is_called()`